tools/runstrat/tests/test_pipeline.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+import os
+
+from runstrat import Args, entrypoint, RunMode
+from src.psstrategy import BasePSStrategy
+
+ARTIFACTS_DIR = "artifacts"
+os.makedirs(ARTIFACTS_DIR, exist_ok=True)
+
+
+def test_pipeline_with_mock_data():
+    args = Args(
+        strategy=BasePSStrategy(name="ExecutionTreeContributedCoverage"),
+        timeout=100,
+        pysymgym_path=Path("../..").absolute(),
```

Comment:
`Path.resolve()` was used before, this change is mainly for an API consistency
```suggestion
        pysymgym_path=Path("../..").resolve(),
```